WORKSPACE

```diff
@@ -169,15 +102,29 @@ go_repository(
 
 # Start using distroless base
 container_pull(
-    name = "distroless_base",
-    digest = "sha256:7fa7445dfbebae4f4b7ab0e6ef99276e96075ae42584af6286ba080750d6dfe5",
+    name = "distroless_base_amd64",
+    digest = "sha256:e7fa8d9d08846d634b16d4ac7d8ecac36b412bfeb1a13ce061183ff020551ba8",
+    registry = "gcr.io",
+    repository = "distroless/base",
+)
+
+container_pull(
+    name = "distroless_base_arm64",
+    digest = "sha256:c60be29941a0be6f748c8cf2e42832f95e9b73276042d3c44212af7cf4a152c9",
+    registry = "gcr.io",
+    repository = "distroless/base",
+)
+
+container_pull(
+    name = "distroless_base_debug_amd64",
```

Comment:
where are these debug images used?

Reply:
These are not being used, but are there for easier debugging during development.
They were there before, so this is the main reason for keeping them.